exspy/signals/eels.py

```diff
@@ -622,10 +620,10 @@ def estimating_function(data, threshold=None):
                     if binned:
                         return data.sum()
                     else:
-                        from scipy.integrate import simps
+                        from scipy.integrate import simpsom
```

Comment:
The integration rule should be `simpson` and not `simpsom`!
Obviously, the line is not included in tests if this did not yield a failure.